examples/Pyplot.ipynb

```diff
@@ -94,6 +94,37 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "### Horizontal and Vertical Lines"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "plt.hline()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "plt.vline()\n",
```

Comment:
@dmadeka Added this. I will make the change you suggested to be able to do `plt.hline(10)` instead of `plt.hline(level=10)`